src/GraphQL/Types/Scalars/BigIntGraphType.cs

```diff
@@ -24,10 +24,13 @@ public class BigIntGraphType : ScalarGraphType
         /// <inheritdoc/>
         public override bool CanParseLiteral(IValue value) => value switch
         {
-            BigIntValue _ => true,
-            LongValue _ => true,
             IntValue _ => true,
+            LongValue _ => true,
+            BigIntValue _ => true,
             _ => false
         };
+
+        /// <inheritdoc/>
+        public override IValue ToAST(object value) => new BigIntValue((BigInteger)ValueConverter.ConvertTo(value, typeof(BigInteger)));
```

Comment:
Remember about eliminating `ValueConverter` in scalars.

Reply:
#2320